resources/config-defaults.edn

```diff
@@ -95,5 +95,10 @@
  ;; on an application, starting from the first submission.
  :application-deadline-days nil
 
+ ;; List of commands to disable. See the swagger schema of
+ ;; /api/applications/{id} field application/permissions for possible
```

Comment:
Hmm, in my swagger, it shows only one possible permission in the example value for that endpoint:

![image](https://user-images.githubusercontent.com/49641103/72324130-e870b300-36b2-11ea-97fa-52cba440abae.png)

Has that particular example application been approved, or what is the cause for that? In any case, I couldn't get a complete list of possible values of :disable-commands from there.

Also nit: the api endpoint is listed as `/api/applications/{application-id}` (instead of `/api/applications/{id}`) in swagger.

Reply:
Check the "Model", not the "Example value". The example value for lists of enum values seems to be a list of length 1.